Python/compile.c

```diff
@@ -2539,36 +2550,23 @@ static int
 compiler_if(struct compiler *c, stmt_ty s)
 {
     basicblock *end, *next;
-    int constant;
     assert(s->kind == If_kind);
-    end = compiler_new_block(c);
+    next = end = compiler_new_block(c);
     if (end == NULL)
         return 0;
 
-    constant = expr_constant(s->v.If.test);
-    /* constant = 0: "if 0" Leave the optimizations to
-     * the pephole optimizer to check for syntax errors
-     * in the block.
-     * constant = 1: "if 1", "if 2", ...
-     * constant = -1: rest */
-    if (constant == 1) {
-        VISIT_SEQ(c, stmt, s->v.If.body);
-    } else {
-        if (asdl_seq_LEN(s->v.If.orelse)) {
-            next = compiler_new_block(c);
-            if (next == NULL)
-                return 0;
-        }
-        else
-            next = end;
-        if (!compiler_jump_if(c, s->v.If.test, next, 0))
+    if (asdl_seq_LEN(s->v.If.orelse)) {
+        next = compiler_new_block(c);
+        if (next == NULL)
```

Comment:
PEP7

Reply:
It is not a new code, I just removed some code and reduced an indentation. `return 0;` without braces is uses every where in this file, so I think that breaking uniformity in the file can add more harm than disobeying this particular part of PEP 7.

Reply:
I agree, disregard the PEP7 comments then.